gusto/configuration.py

```diff
@@ -48,6 +48,8 @@ class OutputParameters(Configuration):
     steady_state_error_fields = []
     #: List of fields for computing perturbations
     perturbation_fields = []
+    #: Dictionary of fields and points at which to dump them
+    point_data = {}
```

Comment:
A couple of reasons:
1) I think it is marginally easier for the user to specify the points as `list_of_xcoords`, `list_of_ycoords` (for example)
2) If the user gave an array of points I would have to think about numpy arrays when setting up the spatial dimensions in the netcdf file - this is definitely doable but my brain was starting to hurt at this point and lists seemed easier.

Happy to change if you think it best...